tests/env/start-swift-ceph.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
Yes, and this is the one which actually started working without `Error: kernel module loop not loaded. Needed by docker image xenopathic/ceph-keystone`, lets see if this is correct.